src/components/Instructions.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import { Card, Typography, Image } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import "../styles/Instructions.css";
+import { LeftOutlined, RightOutlined } from "@ant-design/icons";
+const stepOneImg = require("../assets/images/instructionsStart.JPG");
+const stepTwoImg = require("../assets/images/instructionsPickCard.JPG");
+
+interface InstructionsProps {
+  visible: boolean;
+  hide: () => void;
+}
+
+const instructionSteps: { [key: number]: any[] } = {
```

Comment:
instead of using any[]. you can define a Step type

type Step {
description: HTMLElement,
image: HTMLElement,
}

and then use it instead of any by using { [key: number]: Step[] }

Sorry to be super anal about this but seeing any makes me very sad when using typescript haha

Reply:
All good, I am kind of new to Typescript, so prob will make heaps of mistakes like this.

Is there a way to cast this to string when passing it as an argument for src for the <Image>? I am getting this error after implementing suggested changes
![image](https://user-images.githubusercontent.com/42598015/93014915-14a94e80-f609-11ea-8694-797e04cd3ccd.png)

Reply:
Nvm, I got it :)